docs/comunidad/comunidades.md

```diff
@@ -1,56 +1,42 @@
-<div align="center">
-  <img src="/images/logo.png" alt="Python CDMX Logo">
-</div>
-
+---
+title: Alianzas y Comunidades
 ---
 
-## <i class="fas fa-users"></i> Comunidades Amigas
+![logo PythonCDMX](/images/logo.png){ width=150px .hero-logo }
 
-### <i class="fas fa-star"></i> Comunidades Python en México
+# Comunidades
 
-<div class="communities-grid">
+## :fontawesome-solid-users: Comunidades Amigas
 
-<!-- PyMTY-->
-<a href="https://social.org.mx/@pymty" target="_blank" rel="noopener noreferrer">
-  <div class="community-card">
-    <img src="/images/comunidades/pymty.png" alt="Logo PyMTY">
-    <h4><i class="fab fa-python"></i>PyMTY</h4>
-    <p>Nuevo Leon</p>
-  </div>
-</a>
+### :fontawesome-solid-star: Comunidades Python en México
 
-<!-- Pythonistas GDL -->
-<a href="https://pythonistas-gdl.org/" target="_blank" rel="noopener noreferrer">
-  <div class="community-card">
-    <img src="/images/comunidades/pythonistas-GDL.jpg" alt="Logo PythonistasGDL">
-    <h4><i class="fab fa-python"></i>Pythonistas GDL</h4>
-    <p>Jalisco</p>
-  </div>
-</a>
+<div class="grid cards" markdown>
```

Comment:
La clase `grid` crea un CSS Grid automaticamente (esto se encarga del tema de Responsive)

La clase `cards` les agrega un container `li`, y ya venía por default con _Material for MKDocs_